modules/auxiliary/scanner/http/git_scanner.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
```

Comment:
I don't think Wmap is being maintained so I'd recommend to not use it. Just don't include the mixin, neither use its methods, please.